conference.js

```diff
@@ -3116,12 +3116,12 @@ export default {
     },
 
     /**
-     * Sends a facial expression as a string and its duration as a number
-     * @param {object} payload - Object containing the {string} facialExpression
+     * Sends a face expression as a string and its duration as a number
+     * @param {object} payload - Object containing the {string} faceExpression
      * and {number} duration
      */
-    sendFacialExpression(payload) {
-        room.sendFacialExpression(payload);
+    sendFaceExpression(payload) {
```

Comment:
Where is this function used? This is the legacy conference object and I everything else face-landmark related is built with React, I don't see this used anywhere. Am I missing something?

Reply:
https://github.com/jitsi/jitsi-meet/pull/11307/files#diff-3df1b963ab402f0e79ffdeaacca3881791fbedc759fdce6d7fb37353c23f57abR76

Reply:
nevermind, this is the store conference